manifests/plugin/discovery.pp

```diff
@@ -33,15 +33,19 @@
   if $install_images {
     include ::tftp::params
 
-    foreman::remote_file {"${::tftp::params::root}boot/${kernel}":
-      remote_location => "${source}${kernel}",
+    foreman::remote_file {"${::tftp::params::root}boot/${image_name}":
```

Comment:
<pre>
class foreman::plugin::discovery::params {
  include tftp::params
  $tftp_root = $tftp::params::root
}
</pre>

and here:

<pre>
class foreman::plugin::discovery (
  $tftp_root = $foreman::plugin::discovery::params::tftp_root,
)
</pre>